### ISSUE TITLE
Fine-tuning with a different loss function

### ISSUE THREAD

**Issue by vishalchoudhari11** [ibm-granite/granite-tsfm#319]
Hi there!

Thank you for the amazing work with TTMs. Just a quick question. If I wish to fine-tune using MAE loss function instead of MAE, how do I go about doing that?

**Comment by vishalchoudhari11** [comment]
Just saw that the it accepts loss as an argument.
